apps/sms/test/marionette/lib/views/conversation/accessors.js

```diff
@@ -65,7 +75,9 @@ ConversationAccessor.prototype = {
   },
 
   findMessage: function(id) {
-    return this.client.findElement('.message[data-message-id="' + id + '"]');
+    return this.client.findElement(
+      '.message[data-message-id="' + id + '"] .bubble'
+    );
```

Comment:
ahah, see https://github.com/mozilla-b2g/gaia/pull/31211/files#diff-6692a94bffbe100d8db6802a98b50fb8R66 ;)
I'd be happy if I don't need this findBubble :)